src/commands/token/refreshTokenCommand.ts

```diff
@@ -0,0 +1,35 @@
+import { RefreshTokenInterface } from "../../interfaces/token";
+
+export default class RefreshTokenCommand implements RefreshTokenInterface {
```

Comment:
for all of our classes, I don't think we need the implements. the class already brings all the type safety we need and the interface doesn't add any, it's not used anywhere else either. When we need to verify if a value is of type RefreshTokenCommand we just need to use the class as a type, instead of using the underlying interface.